src/presentation/controllers/v1/state-machine-controller.ts

```
@@ -0,0 +1,302 @@
+    {
+      schema: {
+        description: 'Create a state machine',
+        tags: ['State Machines'],
+        body: CreateStateMachineRequestBodySchema,
+        response: {
+          201: {},
+          409: {},
+        },
+      },
+    },
+    async (request, reply) => {
```

ESLint

[error] 'reply' is defined but never used.

(@typescript-eslint/no-unused-vars)